ts/Core/Chart/Chart3D.ts

```diff
@@ -368,10 +368,7 @@ namespace Chart3D {
          * Whether it is a 3D chart.
          */
         chartProto.is3d = function (): boolean {
-            return Boolean(
-                this.options.chart.options3d &&
-                this.options.chart.options3d.enabled
-            ); // #4280
+            return !!this.options.chart.options3d?.enabled;
```

Comment:
I think we're not using double negation anywhere. Wouldn't it be better to use a comparison to `true` or `|| false`?

Reply:
I can find many `!!(` in the code. My code change is 1 to 1 when it comes to the results. Changing to any of the 2 suggested will still be correct according to the docs and TS but might give different results for non-standard usage.